test/test_repeat.js

```diff
@@ -14,7 +14,7 @@ var ONE_HOUR = 60 * ONE_MINUTE;
 var ONE_DAY = 24 * ONE_HOUR;
 var MAX_INT = 2147483647;
 
-describe('repeat', function() {
+describe.only('repeat', function() {
```

Comment:
remove .only

Reply:
I did it as soon as I realized in [8ae7a58](https://github.com/OptimalBits/bull/pull/944/commits/8ae7a589fff078e8f77c3e06e378cca21a5e9444).